### DISCUSSION TITLE
UI/UX changes

### DISCUSSION THREAD

**Discussion by WTobor** [DevAdventCalendar/DevAdventCalendar#363] [Ideas]
Like any web application, the UI is an essential part of the DevAdventCalent project. This thread is about UI / UX and changes in these areas.
There are many possibilities, but at the beginning, we need to need to determine whether we want to change the entire graphic design, or stay with the current one and refine it, e.g. by improving responsiveness

We already have a bug with responsiveness on Mozilla #248

**Comment by asiajj** [comment]
Hi! I was thinking that there is no need to change the whole design. I would focus on improving the current project and maybe adding new features in the future

**Reply by WTobor** [reply]
Great ;) Do you want only to review the app, or do you want to make improvements in code, too? Both ways are fine for me :) Or maybe you know someone that could help?
We are using Issues to track features and bugs. There is already created a template, so you can create new Issues by filling the form. Let me know if you need any help or more information in that area

**Reply by asiajj** [reply]
Well I can't code so I don't think I would be useful at all in that matter. I'll try to think of some ways to improve the design/user experience and I'll get back to you. Does it sound ok?

**Reply by WTobor** [reply]
Yes please :) First we can find ways to improve design/user experience. Then I will find someone to fix that in code 😄
